lib/orbit_web/auth/guardian.ex

```diff
@@ -0,0 +1,44 @@
+defmodule OrbitWeb.Auth.Guardian do
+  use Guardian, otp_app: :orbit
+
+  # Guardian expects `resource` to be a full user object.
+  # We don't really have that, so use email address for both id and resource
```

Comment:
As I understand it from the second of the two code snippets in [this section of the Guardian docs](https://hexdocs.pm/guardian/introduction-implementation.html#basic-setup), the "resource" can basically be whatever struct or other piece of data represents a "user" in your system, so it's really up to us to define what that should be. It's still a pretty simple example, but it might be worth comparing to [how Skate does it](https://github.com/mbta/skate/blob/bacc5520a66f441a3c60772cfbae0611379148a1/lib/skate_web/auth_manager.ex#L30).

Reply:
@lemald What concern led to the introduction of an additional/separate resource prefix for keycloak? It looks like the Skate team was concerned about what would happen around the transition, but I don't us having issues there (a Guardian expiry would trigger a Keycloak/EntraID new session, yes?).

Reply:
You're correct that it would force users to log back in with the new SSO provider. I think it was just a general precautionary measure, see mbta/skate#2400 and the description I wrote there.

The one concrete case I would worry about would be the logout button - what happens to someone who logged in with Cognito when they click logout?

Reply:
In Glides or Orbit? No Cognito here; in Glides, I assume we'll leave the [Cognito logout case](https://github.com/mbta/glides/blob/main/lib/glides_web/controllers/auth_controller.ex#L123) present for a little while after keycloak is defaulted.

Reply:
Yes, sorry, because you were talking about the transition I thought you were referring to Glides. It isn't really relevant for Orbit at all.

Reply:
Ah, np, yeah I was referring to Skate's transition. Sounds good 👍